test/functional/feature_pruning.py

```diff
@@ -143,6 +143,10 @@ def test_invalid_command_line_options(self):
             extra_args=['-prune=550', '-reindex-chainstate'],
         )
 
+    def test_rescan_blockchain(self):
+        self.restart_node(0, ["-prune=550"])
+        assert_raises_rpc_error(-1, "Can't rescan beyond pruned data. Use RPC call getblockchaininfo to determine your pruned height.", self.nodes[0].rescanblockchain)
```

Comment:
This fails for me to run without wallet

```
test_framework.authproxy.JSONRPCException: Method not found (-32601)

Reply:
Yes, I missed that. `test_rescan_blockchain` should be called if wallet is compiled. Done.